app/models/hackbot/conversations/help.rb

```diff
@@ -0,0 +1,25 @@
+module Hackbot
+  module Conversations
+    class Help < Hackbot::Conversations::Channel
+      HELP_MENU = "Hi! I'm Hack Club's friendly robotic helper. I don't do "\
+                  "much right now, but if you're a club leader you've "\
+                  'probably received some messages from me about the '\
+                  'attendance of your club.'\
+                  "\n\n"\
```

Comment:
Probably a bit more idiomatic to have one newline at the end of line 7 and then the second one on this line (line 8).

That way it's clear where the line breaks are.